### ISSUE TITLE
Django 2.0 compatibility

### ISSUE THREAD

**Issue by eriktelepovsky** [ionelmc/django-redisboard#32]
```
  File "/Users/erik/env/swida/lib/python3.5/site-packages/redisboard/admin.py", line 5, in <module>
    from django.core.urlresolvers import reverse
ImportError: No module named 'django.core.urlresolvers'
```

**Comment by eriktelepovsky** [comment]
Fix:

```
try:
    from django.urls import reverse
except:
    from django.core.urlresolvers import reverse
```

**Comment by eriktelepovsky** [comment]
I created a PR #33 

Check it out, please ;)

**Comment by Fabitosoft** [comment]
Is there any way to get this last upgrade? The current version does not have this fix.

**Comment by eriktelepovsky** [comment]
Install it from the repository instead of pip until author releases new pypi package.

**Comment by ionelmc** [comment]
Released 4.0.
